src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -61,37 +62,33 @@ and prevent the default behavior.
 By preventing a page transition,
 you can retain the credential information while verifying its authenticity.
 
-## Authenticate by sending a request via AJAX
+## Authenticate by sending a request
 
-To authenticate the user, send a request via AJAX and respond with profile information,
-for example, in JSON format.
+To authenticate the user, deliver credential information to your server via `fetch()`.
```

Comment:
This doesn't have to be `fetch()`. The important part is that auth is done through AJAX so that the website can retain credential information to store after a successful authentication. How about this?
```
To authenticate the user, deliver credential information to your server using AJAX.
```